packages/Kokkos/src/DTK_LinearBVH.cpp

```diff
@@ -0,0 +1,69 @@
+#include <details/DTK_DetailsAlgorithms.hpp>
+#include <details/DTK_DetailsTreeConstruction.hpp>
+#include <details/DTK_DetailsTreeTraversal.hpp>
+
+#include <DTK_LinearBVH.hpp>
+
+#include <Kokkos_ArithTraits.hpp>
+
+namespace DataTransferKit
+{
+
+BVH::BVH( AABB const *bounding_boxes, int n )
+    : _leaf_nodes( "leaf_nodes", n )
+    , _internal_nodes( "internal_nodes", n - 1 )
+    , _indices( "sorted_indices", n )
+{
+    // determine the bounding box of the scene
+    Details::calculateBoundingBoxOfTheScene( bounding_boxes, n,
+                                             _internal_nodes[0].bounding_box );
```

Comment:
OK to rename.  I don't like `Global` because this is only for your "local" MPI rank.